sklearn/ensemble/_stacking.py

```diff
@@ -547,7 +547,7 @@ class StackingClassifier(ClassifierMixin, _BaseStacking):
 
     n_features_in_ : int
         Number of features seen during :term:`fit`. Only defined if the
-        underlying classifier exposes such an attribute when fit.
+        underlying estimator exposes such an attribute when fit.
```

Comment:
This is the kind I'm not sure of. In this case it's not really important, but it's making the text technically less accurate.

I guess we can accept these changes for the benefit of having them tested to be the same.

Reply:
Yup, I had exactly the same thought! What pushed me over is that 'estimator' is used a few times elsewhere in both `StackingRegressor` and `StackingClassifier` docstring. 

Now that I look at it more `estimator` is more accurate for `StackingClassifier`, since:

https://github.com/scikit-learn/scikit-learn/blob/153205a87747e0c31cca411fa44f2090234cd69b/sklearn/ensemble/_stacking.py#L457-L464 

though for `StackingRegressor` it would be 'regressor'.

I'm still not 100% on this though and happy to change!

Reply:
I think I'm happy to go with estimator here but this is exactly showing that we might need a way to be more lenient in some cases. But here, more lenient could be that we only want to check that the parameter and types are fine and ignore the docstring.

Reply:
To clarify, check the types only and ignore the parameter description?

Reply:
Yes or otherwise, passing a regex for the matching.